shell/pages/c/_cluster/manager/jwt.authentication/index.vue

```diff
@@ -159,7 +160,7 @@ export default {
             if (!configName) {
               const clusterProxyConfig = await this.$store.dispatch('management/create', {
                 enabled:  true,
-                metadata: { namespace: id, generateName: 'cluster-proxy-config-' },
+                metadata: { namespace: id, name: 'cluster-proxy-config' },
```

Comment:
as a general rule name should be unique within a namespace. as namespace is a cluster this should be safe, though it feels wrong. was there a reason to move to a static name?

Reply:
I've checked https://confluence.suse.com/pages/viewpage.action?pageId=1333068261#ServiceAccountTokenAuthentication(Vault)-Testing
And there under 3a they are saying " The only way to enable it is to create a clusterproxyconfig object named "clusterproxyconfig" in the namespace of the downstream cluster." so I've changed it to match this spec in case it will be enforced later on.

Reply:
Ahh makes sense. Probably needs to be without the hyphens as well.

Gave the spec a quick check and it seems not everyone has access to them, though we always show the menu item and list (it'll be empty and the following error will be in console ` Error in fetch(): Error: Unknown schema for type: management.cattle.io.clusterproxyconfig`). Can you create a 2.10 issue to hide the menu item if the user has no permissions to see the resource?

Reply:
Created https://github.com/rancher/dashboard/issues/11461